onnxruntime/core/providers/cpu/tensor/split.cc

```diff
@@ -17,6 +17,7 @@ ONNX_CPU_OPERATOR_KERNEL(
                                       std::vector<MLDataType>{
                                           DataTypeImpl::GetTensorType<float>(),
                                           DataTypeImpl::GetTensorType<double>(),
+                                          DataTypeImpl::GetTensorType<int>(),
```

Comment:
use one of the exact int types that you want to support.

Reply:
done; support int32_t and int64_t as python int can be these two dependent on environment